packages/ovm-toolchain/test/test-waffle-v2/core/precompiles.spec.ts

```diff
@@ -12,21 +12,17 @@ import { waffleV2 } from '../../../src/waffle/waffle-v2'
 /* Contract Imports */
 import * as Precompiles from '../../temp/build/waffle/Precompiles.json'
 
-const overrides = {
-  gasLimit: 20000000,
-}
-
 describe('Precompile Support', () => {
   let wallet: Wallet
   let provider: any
   beforeEach(async () => {
-    provider = new waffleV2.MockProvider(overrides)
+    provider = new waffleV2.MockProvider()
```

Comment:
Ah so this removes an unneeded `overrides` field which is no longer needed because we're returning a hardcoded value in our Ganache

Reply:
That right?

Reply:
Sooort of, yes. Our Ganache just replaces the gas limit of each transaction with the max gas limit.